app/services/manifest_builder.rb

```diff
@@ -323,6 +325,12 @@ def geo_image?(member)
       end
   end
 
+  class NumismaticIssueNode < CollectionNode
+    def members
+      @members ||= super.select { |x| x.member_ids.present? }
```

Comment:
Could you replace `x` with a more specific variable name? Will a member of an issue always be a coin? If so maybe use `coin`. Also, it would be nice to have a comment here, like "Only include coins which have file sets; otherwise there's no image for a viewer" or something.

Reply:
@hackmastera I updated.